### PR TITLE
feat: filter and sorting

### PR DESCRIPTION
**Pull request by devsurajrai**
Added filter and sorting feature for the product listing page. Please review and provide any suggestions that you might have

Screenshot 2022-03-26 at 12 45 22 PM
.

### CODE REVIEW

File: src/components/ProductListingMain.component/product-listing-main.css [hunk 1]
```diff
@@ -12,7 +12,6 @@
     grid-template-columns: minmax(0, 1fr) minmax(0, 5fr);
     grid-template-rows: repeat(minmax(0, 1fr));
     grid-column-gap: 1rem;
-    column-gap: 1rem;
 
-    overflow-y: hidden;
+    /* overflow-y:hidden; */
```
Comment (palash23jain) [inline src/components/ProductListingMain.component/product-listing-main.css:16]: commented code can be removed

**Review by palash23jain — APPROVED** [review]
Clean code

File: src/components/HomeMain.component/Categories.component/HomeCategories.jsx [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
     <div className="cat-card br-sm">
-      <img className="res-img" src={racket} alt="" />
-      <a className="button btn-sm cat-card-btn text-sm" href="">
-        RACQUETS
-      </a>
+      <img className="res-img" src={category.categoryImgUrl} alt="" />
+      <Link
+        to={{
+          pathname: "/products",
+          search: `?category=${category.categoryName}`,
```
Comment (varnita619) [inline src/components/HomeMain.component/Categories.component/HomeCategories.jsx:12]: instead of repeatedly using category you can destructure that

**Review by varnita619 — APPROVED** [review]

File: src/components/HomeMain.component/Categories.component/HomeCategories.jsx (hunk 2 above)
Comment (devsurajrai) [reply src/components/HomeMain.component/Categories.component/HomeCategories.jsx:12]: got it.

File: src/components/ProductListingMain.component/product-listing-main.css (hunk 1 above)
Comment (devsurajrai) [reply src/components/ProductListingMain.component/product-listing-main.css:16]: will do that

**Pull request merged**